src/style.css

```diff
@@ -1,3 +1,73 @@
+:root {
+  --text-color: #000;
+  --text-color-active: #00acdc;
+  --header-height: 60px;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.container {
+  margin: 0 auto;
+  max-width: 1200px;
+  padding: 0 50px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: var(--header-height);
+  position: relative;
+}
+
+.header_nav__list {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  height: var(--header-height);
+}
+
+.header_nav__item {
+  position: relative;
+  display: flex;
+  align-items: center;
```

Comment:
extra code, remove please

Reply:
@StasSokolov  if I remove this code, I have 4 failed in test https://prnt.sc/5npSmf5KepuJ